Remotion/Web/Development.WebTesting.IntegrationTests/AnchorControlObjectTest.cs

```diff
@@ -139,10 +139,6 @@ public void TestClick ()
       var htmlAnchor = home.Anchors().GetByLocalID("MyHtmlAnchor");
       home = htmlAnchor.Click().Expect<WxePageObject>();
       Assert.That(home.Scope.FindId("TestOutputLabel").Text, Is.EqualTo("MyHtmlAnchor"));
-
-      var htmlAnchorWithJavaScriptLink = home.Anchors().GetByLocalID("MyHtmlAnchorWithJavaScriptLink");
```

Comment:
Please try, I think we can solve this by appending `return false;` to the onclick in the APSX file. That was missing from the get-go and only now popped up as a problem.

` <a id="MyHtmlAnchorWithJavaScriptLink" href="#" onclick="javascript:myClickHandler('MyHtmlAnchorWithJavaScriptLink'); return false;" runat="server">MyHtmlAnchorWithJavaScriptLink</a>`

Reply:
Please re-check: right now, the a-tag has both a href and an onclick-handler. since the onclick-handler does not end with a `return false;` statement, clicking the anchor will first execute the onlick handler and then the href. As far as I understand the problem, executing the href with `#` will break the test. By appending `return false;` this behavior should change (unless I'm mistaken).